test-harness/specs/content-negotiation/E2E-Accept-Content-Negotiation-AC-7.oas2.txt

```diff
@@ -3,17 +3,19 @@ When I send a request to an operation with Accept header containing */*
 And this operation produces a response with content-type matching */* (text/plain)
 Then I should get back a response with Content-Type of text/plain
 ====spec====
-swagger: "2.0"
+swagger: '2.0'
 paths:
   /todos:
     get:
       produces:
         - text/plain
       responses:
-        200:
+        '200':
           description: Get Todo Items
           examples:
             '*/*': hello
+          schema:
+            type: string
```

Comment:
this spec was invalid (needed the schema prop) so was causing this test to fail since the new http-spec wasn't parsing it because it was invalid.